changelog/_unreleased/2025-01-29-mark-import-export-internal.md

```diff
@@ -0,0 +1,31 @@
+---
+title: mark Import/Export internal
+issue: NEXT-40446
+---
+# Core
+* Added `Shopware\Core\Content\ImportExport\Event\ImportExportAfterProcessFinishedEvent.php`
+* Deprecated `Shopware\Core\Content\ImportExport\Event\ImportExportAfterImportRecordEvent.php` which will be removed in 6.7.0.0 without replacement
```

Comment:
also here and on the other occurrences here. do not reference the file, but he full class name instead